src/tests/integration/test-data/foundry-prove-skip-legacy

```diff
@@ -126,32 +143,40 @@ LoopsTest.test_sum_100()
 LoopsTest.test_sum_1000()
 LoopsTest.testSumToNBroken(uint256)
 LoopsTest.testSumToN(uint256)
-MockCallTest.testMockCall()
```

Comment:
Can you time this proof on the legacy backend and skip it if it takes longer than 300s?